force-app/test/jest-mocks/lightning/navigation.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * For the original lightning/navigation mock that comes by default with
```

Comment:
Will the default mock that's shipped with `lwc-jest` make this obsolete in the future? Or is this an enhancement to the shipped mock that'll go eventually into `lwc-jest`?

Reply:
This is an enhancement to what's shipped with `lwc-jest`. I don't plan on adding these changes to the default mock unless there's strong support by others and customers to do so.